modules/react/loading-dots/lib/LoadingDots.tsx

```diff
@@ -20,41 +15,48 @@ const keyframesLoading = keyframes({
   },
 });
 
-const loadingDotStyles = createStyles({
-  backgroundColor: base.soap400,
-  width: system.space.x4,
-  height: system.space.x4,
-  fontSize: system.space.zero,
-  borderRadius: system.shape.round,
-  transform: 'scale(0)',
-  display: 'inline-block',
-  animationName: keyframesLoading,
-  animationDuration: `${ANIMATION_DURATION_MS}ms`,
-  animationIterationCount: 'infinite',
-  animationTimingFunction: 'ease-in-out',
-  animationFillMode: 'both',
-  '&:nth-child(1)': {
-    animationDelay: '0ms',
-  },
-  '&:nth-child(2)': {
-    animationDelay: `calc(${ANIMATION_DURATION_MS}ms * (4/35))`, // 160
-  },
-  '&:nth-child(3)': {
-    animationDelay: `calc(${ANIMATION_DURATION_MS}ms * (8/35))`, // 320
+const loadingDotStencil = createStencil({
+  vars: {
+    animationDurationMs: '40ms',
```

Comment:
we don't have tokens for animation or `duration` right?

Reply:
I don't think we do have those tokens

Reply:
no, we don't. I don't think we have it in our roadmap soon

Reply:
all good I was just curious if that becomes some type of token? Maybe a larger discussion around animation!